### PR TITLE
feat(ocsp_crl): add global options for CRL and OCSP

### PR DESCRIPTION
**Pull request by thalesmg**

### CODE REVIEW

File: src/i18n/Settings.js [hunk 1]
```diff
@@ -117,6 +117,17 @@ export default {
     zh: '如果未加载身份验证插件，则默认情况下允许匿名身份验证。建议在生产部署中禁用该选项！',
     en: 'Allow anonymous authentication by default if no auth plugins loaded. Disable the option in production deployment',
   },
+  crl_cache_http_timeout: {
+    zh: `获取CRLs时HTTP请求的超时。这对所有听众来说是全球性的。`,
```
Comment (zmstone) [inline src/i18n/Settings.js:121]:
```suggestion
    zh: `获取 CRL 时 HTTP 请求的超时限制。该参数适用于所有监听器。`,
```

File: src/i18n/Settings.js [hunk 2]
```diff
@@ -117,6 +117,17 @@ export default {
...
+    en: `The timeout for the HTTP request when fetching CRLs.  This is global
+         for all listeners.`,
+  },
+  crl_cache_refresh_interval: {
+    zh: `从服务器刷新CRL的周期。 这是全球
+         所有URL和监听器。 不能短于1分钟。`,
```
Comment (zmstone) [inline src/i18n/Settings.js:127]:
```suggestion
    zh: `从 CRL 服务器刷新 CRL 的周期。 该配置会被所有启用 CRL 检查的监听器用到。 不能短于1分钟。`,
```

File: src/i18n/Settings.js [hunk 3]
```diff
@@ -117,6 +117,17 @@ export default {
...
+    en: `The period to refresh the CRLs from the servers.  This is global
+         for all URLs and listeners.  Cannot be shorter than 1 minute.`,
```
Comment (zmstone) [inline src/i18n/Settings.js:129]:
```suggestion
    en: `The period to refresh the CRLs from the servers.  This is global
         for all listeners with CRL check enabled.  Cannot be shorter than 1 minute.`,
```

File: src/i18n/Settings.js [hunk 4]
```diff
@@ -117,6 +117,17 @@ export default {
     zh: '如果未加载身份验证插件，则默认情况下允许匿名身份验证。建议在生产部署中禁用该选项！',
     en: 'Allow anonymous authentication by default if no auth plugins loaded. Disable the option in production deployment',
   },
+  crl_cache_http_timeout: {
+    zh: `获取CRLs时HTTP请求的超时。这对所有听众来说是全球性的。`,
+    en: `The timeout for the HTTP request when fetching CRLs.  This is global
+         for all listeners.`,
```
Comment (zmstone) [inline src/i18n/Settings.js:123]:
```suggestion
    en: `The timeout for the HTTP request when fetching CRLs. This is a global setting for all listeners with CRL check enabled.`,
```

File: src/i18n/Settings.js [hunk 5]
```diff
@@ -117,6 +117,15 @@ export default {
     zh: '如果未加载身份验证插件，则默认情况下允许匿名身份验证。建议在生产部署中禁用该选项！',
     en: 'Allow anonymous authentication by default if no auth plugins loaded. Disable the option in production deployment',
   },
+  crl_cache_http_timeout: {
+    zh: `获取 CRL 时 HTTP 请求的超时限制。该参数适用于所有启用了 CRL 检查的监听器。`,
+    en: `The timeout for the HTTP request when fetching CRLs. This is a global setting for all listeners with CRL check enabled.`,
```
Comment (ysfscream) [inline src/i18n/Settings.js:122]:
```suggestion
    zh: '获取 CRL 时 HTTP 请求的超时限制。该参数适用于所有启用了 CRL 检查的监听器。',
    en: 'The timeout for the HTTP request when fetching CRLs. This is a global setting for all listeners with CRL check enabled.',
```

We need not use the JavaScript template syntax. Just use single quotes.

File: src/i18n/Settings.js [hunk 6]
```diff
@@ -117,6 +117,15 @@ export default {
...
+  },
+  crl_cache_refresh_interval: {
+    zh: `从 CRL 服务器刷新 CRL 的周期。 该配置会被所有启用 CRL 检查的监听器用到。 不能短于1分钟。`,
+    en: `The period to refresh the CRLs from the servers.  This is global
+         for all listeners with CRL check enabled.  Cannot be shorter than 1 minute.`,
```
Comment (ysfscream) [inline src/i18n/Settings.js:127]:
```suggestion
    zh: '从 CRL 服务器刷新 CRL 的周期。 该配置会被所有启用 CRL 检查的监听器用到。 不能短于1分钟。',
    en: 'The period to refresh the CRLs from the servers. This is global
         for all listeners with CRL check enabled. Cannot be shorter than 1 minute.',
```

same here.

**Review by ysfscream — APPROVED** [review]

**Pull request merged**